landlab/components/landslides/landslide_probability.py

```diff
@@ -45,14 +48,15 @@ class LandslideProbability(Component):
     stability index (Factor of Safety).
 
     The driving force for failure is provided by the user in the form of
-    groundwater recharge; four options for providing recharge are supported.
+    groundwater recharge OR depth to groundwater. Four options for providing 
+    recharge plus four options of depth to groundwater are supported.
     The model uses topographic and soil characteristics provided as input
     by the user.
 
     The main method of the LandslideProbability class is
-    `calculate_landslide_probability()``, which calculates the mean soil
-    relative wetness, probability of soil saturation, and probability of
-    failure at each node based on a Monte Carlo simulation.
+    `calculate_landslide_probability()`, which calculates the mean soil
+    relative wetness, mean water table depth, mean recharge, probability of soil saturation, 
+    and probability of failure at each node based on a Monte Carlo simulation.   
 
```

Comment:
@ChristinaB very excited that you are adding these things! 🎉

Reply:
I'm so excited to have this PR done and start using the component on a non-synthetic unit landscape.